bigtest/interactors/provider-edit.js

```diff
@@ -0,0 +1,53 @@
+import {
+  Interactor,
+  interactor,
+  clickable,
+  property,
+  value,
+  fillable,
+  text,
+  is,
+  attribute,
+  action,
+  isPresent
+} from '@bigtest/interactor';
+
+import Toast from './toast';
+
+@interactor class ProviderEditNavigationModal {
+  cancelNavigation = clickable('[data-test-navigation-modal-dismiss]');
+  confirmNavigation = clickable('[data-test-navigation-modal-continue]');
+}
+
+@interactor class ProviderEditDropDown {
+  clickDropDownButton = clickable('button');
+  isExpanded = attribute('button', 'aria-expanded');
+}
+
+@interactor class ProviderEditDropDownMenu {
+   cancel = new Interactor('.tether-element [data-test-eholdings-provider-cancel-action]');
+}
+
+@interactor class ProviderEditPage {
+  navigationModal = new ProviderEditNavigationModal('#navigation-modal');
+  paneTitle = text('[data-test-eholdings-details-view-pane-title]');
+  name = text('[data-test-eholdings-details-view-name="provider"]');
+  nameHasFocus = is('[data-test-eholdings-details-view-name="provider"]', ':focus');
+  clickCancel= action(function () {
+    return this
+      .dropDown.clickDropDownButton()
+      .dropDownMenu.cancel.click();
+  });
+  clickSave = clickable('[data-test-eholdings-provider-save-button]');
+  isSaveDisabled = property('[data-test-eholdings-provider-save-button]', 'disabled');
+  hasErrors = isPresent('[data-test-eholdings-details-view-error="package"]');
```

Comment:
should this be `="provider"` instead of package?

Reply:
Yes -- thanks good catch

Reply:
Updated and also added unit tests for Provider PUT and GET server errors